.github/workflows/reassurePerformanceTests.yml

```diff
@@ -0,0 +1,50 @@
+name: Reassure Performance Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['**']
+
+jobs:
+  perf-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Setup NodeJS
+        uses: Expensify/App/.github/actions/composite/setupNode@main
+
+      - name: Run performance testing script
+        shell: bash
+        run: |
+          set -e 
+          BASELINE_BRANCH=${BASELINE_BRANCH:="main"}
+          git fetch origin
+          git switch "$BASELINE_BRANCH"
+          npm install --force
+          npx reassure --baseline
+          git switch --detach -
+          npm install --force
+          npx reassure --branch
+
+      - name: Read output.json
+        id: reassure
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: .reassure/output.json
+
+      - name: Validate output.json
+        id: validateReassureOutput
+        uses: ArekChr/App/.github/actions/javascript/validateReassureOutput@feat/reassure
```

Comment:
Just a friendly reminder here, seems like this action from your fork is temporary? @ArekChr

Reply:
Yes, this will be changed to `Expensify/App/.github/actions/javascript/validateReassureOutput@main` before merging, or we can create a separate PR with that workflow to main branch.